README.md

```diff
@@ -18,99 +22,115 @@ We have active, helpful communities on Twitter, Discord, and Telegram.
+endpoint to interact with Ethereum. We recommend using a service such as
+Alchemy or Infura. Larger validators may opt to use any existing full node they
+are already running for other purposes. Setup and configuration of an Ethereum
+node is left as an exercise for the reader. 
+
+The Steward CLI now supports all of the same commands as `gorc` and is the
+recommended way to configure delegate keys for new validators and to run the
+Orchestrator. __The Steward CLI is used to run *both* the `steward` and
+`orchestrator` processes__. There are post-installation steps for the `steward`
+process outlined at the end of the installation steps below. These are required
+for your Steward to participate in the protocol. For more information on these
+setup steps for Steward, see [Validators Instructions for Setting Up
+Steward](https://github.com/PeggyJV/steward/blob/main/docs/02-StewardForValidators.md)
+in the Steward repository.
+
+> NOTE: The Steward CLI and Steward itself are distinct concepts in this
+> document. The Steward CLI is used to start both the `steward` and
+> `orchestrator` processes, while "Steward" refers specifically to the
+> `steward` process.
+
 ### Installation
 
-```bash 
+```bash
 # Create an installation directory
 mkdir install && cd install
 
-# Install Orchestrator
-wget https://github.com/PeggyJV/gravity-bridge/releases/download/v0.3.9/gorc && chmod +x * && sudo mv * /usr/bin
-
-# Install Geth
-wget https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.10.4-aa637fd3.tar.gz && tar -xvf geth-linux-amd64-1.10.4-aa637fd3.tar.gz && sudo mv geth-linux-amd64-1.10.4-aa637fd3/geth /usr/bin/geth && rm -rf geth-linux-amd64-1.10.4-aa637fd3*
+# Install Steward
+wget https://github.com/PeggyJV/steward/releases/download/v3.4.2/steward \
+    && chmod +x * \
+    && sudo mv * /usr/bin
 
 # Install Sommelier
-wget https://github.com/PeggyJV/sommelier/releases/download/v3.1.1/sommelier_3.1.1_linux_amd64.tar.gz && tar -xf sommelier_3.1.1_linux_amd64.tar.gz && sudo mv sommelier /usr/bin && rm -rf sommelier_3.1.1_linux_amd64* LICENSE README.md
+wget https://github.com/PeggyJV/sommelier/releases/download/v6.0.0/sommelier_6.0.0_linux_amd64.tar.gz \
+    && tar -xf sommelier_6.0.0_linux_amd64.tar.gz \
+    && sudo mv sommelier /usr/bin \
+    && rm -rf sommelier_6.0.0_linux_amd64* LICENSE README.md
 
-# Fetch systemd unit files
-wget https://raw.githubusercontent.com/PeggyJV/sommelier/main/contrib/systemd/geth.service https://raw.githubusercontent.com/PeggyJV/sommelier/main/contrib/systemd/gorc.service https://raw.githubusercontent.com/PeggyJV/sommelier/main/contrib/systemd/sommelier.service
+# Fetch systemd unit file examples
+wget \
+    https://raw.githubusercontent.com/PeggyJV/sommelier/main/contrib/systemd/sommelier.service \
+    https://raw.githubusercontent.com/PeggyJV/sommelier/main/contrib/systemd/orchestrator.service \
+    https://raw.githubusercontent.com/PeggyJV/sommelier/main/contrib/systemd/steward.service
```

Comment:
Is there actually a file checked in for `orchestrator.service` and `steward.service`? Otherwise these will be dead links.

Reply:
Ah forgot to pull those over